dnf/dnf-main.c

```diff
@@ -90,7 +120,8 @@ static const GOptionEntry global_opts[] = {
   { "nobest", '\0', G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE, &opt_nobest, "Do not limit the transaction to the best candidates", NULL },
   { "nodocs", '\0', G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE, &opt_nodocs, "Install packages without docs", NULL },
   { "releasever", '\0', G_OPTION_FLAG_NONE, G_OPTION_ARG_CALLBACK, process_global_option, "Override the value of $releasever in config and repo files", "RELEASEVER" },
-  { "setopt", '\0', G_OPTION_FLAG_NONE, G_OPTION_ARG_CALLBACK, process_global_option, "Set transaction flag, like tsflags=nodocs", "FLAG" },
+  { "setopt", '\0', G_OPTION_FLAG_NONE, G_OPTION_ARG_CALLBACK, process_global_option,
+    "Override a configuration option (install_weak_deps=0/1, tsflags=nodocs)", "<option>=<value>" },
```

Comment:
What does mean ``<option>=<value>``? There is no general support for any option.

Reply:
Yes. Only install_weak_deps=0/1 and tsflags=nodocs are supported now. As is noted in description.
Do you have better idea than `<option>=<value>`?  Only `FLAG` is not true any more.

Reply:
I am looking again and again and I belive that the original statement was the best one ``<option>=<value>``